autopush/websocket.py

```diff
@@ -368,12 +368,10 @@ def _check_other_nodes(self, result):
             connected_at = self._get_aval(attrs.get("connected_at", {}), 'N')
             url = "%s/notif/%s/%s" % (node_id, uaid, connected_at)
             if not getattr(self, 'testing', False):
-                d = self.ap_settings.agent.request(
+                self.ap_settings.agent.request(
                     "DELETE",
                     url.encode("utf8"),
-                ).addCallback(self.finish_hello)
-                d.addErrback(self.log_err, extra="Failed to delete old node")
-                return
+                ).addErrback(self.log_err, extra="Failed to delete old node")
```

Comment:
Probably should have two errbacks, one that traps all the connect errors to a dead node, then the self.log_err. Otherwise we'll prolly end up with tons of useless connect errors in sentry again.